libs/mini-rx-store/src/lib/store-core.ts

```diff
@@ -28,12 +32,11 @@ let isStoreInitialized = false;
 export const reducerState = new State<{
```

Comment:
Some code here can be replaced with Reducer Manager. See also the Signal Store store-core.ts for inspiration.

Normally these pieces of code can be replaced with Reducer Manager:

reducerState, reducer$, maybe more.

Reply:
I'm not really sure how to proceed with this. Can you provide an example perhaps of how this would look for `reducerState` and `reducer$`? Am I supposed to extend the reducer manager d